core/domain/interaction_jobs_one_off_test.py

```diff
@@ -48,6 +48,14 @@ def mock_validate(unused_self):
     pass
 
 
+def start_multiple_item_interaction_lt_one_off_job(self, job_class):
+    """Helper function to start MultipleItemInteractionLtOneOffJob."""
+    job_id = job_class.create_new()
+    job_class.enqueue(job_id)
+    self.process_and_flush_pending_mapreduce_tasks()
+    return job_id
+
```

Comment:
Put this function to the `MultipleItemInteractionLtOneOffJobTests` and mark it as private inside (prefix the function name with `_`).

Reply:
Btw it would be useful to rework this to be similar to https://github.com/oppia/oppia/blob/develop/core/domain/activity_jobs_one_off_test.py#L76